app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/pocket/PocketStoriesComposables.kt

```diff
@@ -202,270 +128,96 @@ fun PocketStoriesCategories(
-                }
-                val itemsPositions = IntArray(itemsSizes.size) { 0 }
-                with(arrangement) {
-                    arrange(constraints.maxWidth, itemsSizes, LayoutDirection.Ltr, itemsPositions)
-                }
-
-                itemRow.forEachIndexed { itemIndex, item ->
-                    item.place(
-                        x = itemsPositions[itemIndex],
-                        y = (rowIndex * item.height) + (rowIndex * verticalItemsSpacingPixels)
-                    )
-                }
-            }
-        }
-    }
-}
-
-/**
- * Displays [content] in a layout which will have at the bottom more information about Pocket
- * and also an external link for more up-to-date content.
- */
-@Composable
-fun PocketRecommendations(
-    content: @Composable (() -> Unit)
-) {
-    val annotatedText = buildAnnotatedString {
-        val text = "Pocket is part of the Firefox family. "
-        val link = "Learn more."
-        val annotationStartIndex = text.length
-        val annotationEndIndex = annotationStartIndex + link.length
-
-        append(text + link)
-
-        addStyle(
-            SpanStyle(textDecoration = TextDecoration.Underline),
-            start = annotationStartIndex,
-            end = annotationEndIndex
-        )
-
-        addStringAnnotation(
-            tag = "link",
-            annotation = "https://www.mozilla.org/en-US/firefox/pocket/",
-            start = annotationStartIndex,
-            end = annotationEndIndex
-        )
-    }
+    val link = stringResource(R.string.pocket_stories_feature_learn_more)
+    val text = stringResource(R.string.pocket_stories_feature_caption, link)
+    val linkStartIndex = text.indexOf(link)
+    val linkEndIndex = linkStartIndex + link.length
 
     Column(
-        modifier = Modifier.padding(vertical = 16.dp),
-        horizontalAlignment = Alignment.CenterHorizontally
+        horizontalAlignment = Alignment.CenterHorizontally,
     ) {
-        content()
-
-        CompositionLocalProvider(LocalContentAlpha provides ContentAlpha.medium) {
-            ClickableText(
-                text = annotatedText,
-                style = MaterialTheme.typography.caption,
-                onClick = {
-                    annotatedText
-                        .getStringAnnotations("link", it, it)
-                        .firstOrNull()?.let {
-                            println("Learn more clicked! Should now access ${it.item}")
-                        }
-                }
+        Row(
+            Modifier
+                .fillMaxWidth()
+                .semantics(mergeDescendants = true) { },
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Icon(
+                painter = painterResource(id = R.drawable.pocket_vector),
+                contentDescription = null,
+                // Apply the red tint in code. Otherwise the image is black and white.
+                tint = Color(0xFFEF4056)
```

Comment:
Does this work for both light / dark theme?

Reply:
Tested and it looks okay in both variations.